src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -293,6 +305,35 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
                             .setContentTriples(descriptionTriples);
                     toWrite.add(f6DescId);
 
+                    if (DC_DS.equals(dsId) && !disableDc) {
+                        DC dc = new DC();
+                        try {
+                            dc = DC.parseDC(dv.getContent());
+                        } catch (final IOException e) {
+                            LOGGER.error("problem parsing DC record within: " + f6DsId);
+                            throw new UncheckedIOException(e);
+                        } catch (JAXBException e) {
+                            LOGGER.error("problem parsing DC within: " + f6DsId);
+                        }
```

Comment:
I was curious about trying to create a `JAXBException` here because I wanted to see what would happen even though it should continue to work (this was before I saw the empty DC object being created). If you want though you could use a similar pattern to the one in `parseRdfXml` since it feels like we would want to fail if there's an error parsing the XML.

Reply:
Good point @mikejritter. I've addressed it in the patest push